release_files/Dockerfile.redhat

```diff
@@ -117,6 +117,11 @@ RUN     set -e ; \
         useradd --home-dir /home/ovms --create-home --uid 5000 --gid 5000 --groups 39,44 --shell /bin/bash --skel /dev/null ovms
 
 COPY --from=base_build /ovms /ovms
+
+# Remove capi library
```

Comment:
move it to the first stage of the dockerfile

Reply:
Why?

Reply:
to reduce the final image size

Reply:
Cant move earlier , it just has been copied from /ovms.